src/SFA.DAS.AANHub.Application/Members/Commands/PostMemberStatus/PostMemberStatusCommandValidator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using FluentValidation;
+using SFA.DAS.AANHub.Application.Common.Validators.MemberId;
+using SFA.DAS.AANHub.Application.Common.Validators.RequestedByMemberId;
+using SFA.DAS.AANHub.Domain.Common;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.Members.Commands.PostMemberStatus;
+
+public class PostMemberStatusCommandValidator : AbstractValidator<PostMemberStatusCommand>
+{
+    public const string RequestedByMemberIdNotValid = "Requesting member should be an administrator or a regional chair";
+
+    public PostMemberStatusCommandValidator(IMembersReadRepository membersReadRepository)
+    {
+        Include(new MemberIdValidator(membersReadRepository));
+        Include(new RequestedByMemberIdValidator(membersReadRepository));
+
+        RuleFor(x => x.RequestedByMemberId)
```

Comment:
This validation is already in two places
PutEventGuestsCommentValidator
CalendarEventCommandBaseValidator

I think we should refactor this to make it reusable as we will need this for any actions that admins can take. 

Also this will replace RequestedByMemberIdValidator as it is redundant.

Reply:
this is now done

Reply:
Not in CalendarEventCommandBaseValidator and 'DependentRules' used